<?xml version="1.0"?>

<launch>
  <arg name="output_type"    default="screen"/>
  <arg name="camera_name"    default="none"/>
  <arg name="folder"         default="none"/>
  <arg name="fps"            default="40"/>
  <arg name="show"           default="-2"/>
  <arg name="generate"       default="false"/>
  <arg name="save_3d"        default="false"/>
  <arg name="t_offset"       default="0"/>
  <arg name="t_len"          default="-1"/>

  <node pkg="evimo" name="evimo_vis_offline" type="datagen_offline" output="$(arg output_type)" respawn="false" required="true">
    <param name="folder"             value="$(arg folder)"/>
    <param name="camera_name"        value="$(arg camera_name)"/>
    <param name="fps"                value="$(arg fps)"/>
    <param name="show"               value="$(arg show)"/>
    <param name="generate"           value="$(arg generate)"/>
    <param name="save_3d"            value="$(arg save_3d)"/>
    <param name="start_time_offset"  value="$(arg t_offset)"/>
    <param name="sequence_duration"  value="$(arg t_len)"/>
  </node>

  <!-- Start rviz visualization with preset config -->
  <!--node pkg="rviz" type="rviz" name="rviz" args="-d $(find evimo)/launch/evimo.rviz" required="true"/-->
</launch>
